shortfin/requirements-iree-compiler.txt

```diff
@@ -1,4 +1,4 @@
-# Keep in sync with "ref: iree-" in .github/workflows/* and GIT_TAG in CMakeLists.txt
+# Keep in sync with version in shortfin/buidl_tools/iree_version.txt
```

Comment:
typo in here, but I think we can just say that this should be compatible with `SHORTFIN_IREE_GIT_TAG` in the CMake and have the workflows use FetchContent